Formula/openresty.rb

```diff
@@ -10,17 +10,18 @@ class Openresty < Formula
     url "https://openresty.org/download/openresty-#{VERSION}.tar.gz"
     sha256 "74f076f7e364b2a99a6c5f9bb531c27610c78985abe956b442b192a2295f7548"
 
-    #patch do
+    # patch do
     #  url "https://raw.githubusercontent.com/openresty/openresty/refs/heads/master/patches/LuaJIT2-20241104.patch"
     #  sha256 "5e1f56e32f481cd42c73612af7b6e4a06ee5a6f1f711553a76fb505ca2dfebeb"
-    #end
+    # end
   end
 
   option "with-postgresql", "Compile with ngx_http_postgres_module"
   option "with-iconv", "Compile with ngx_http_iconv_module"
   option "with-slice", "Compile with ngx_http_slice_module"
+  option "without-geoip", "Compile without ngx_http_geoip module"
 
-  depends_on "libmaxminddb"
+  depends_on "libmaxminddb" unless build.without? "geoip"
```

Comment:
brew audit complains about a lot in this file, and it says that `unless` and `without?` would be better as an `if with?` but that would change the default behavior which I think would be a bigger change. (It would not affect me, but I don't know why these particular options were chosen.)